packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_SequencerFeeWallet.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
+
+/* Library Imports */
+import { Lib_PredeployAddresses } from "../../libraries/constants/Lib_PredeployAddresses.sol";
+
+/* Contract Imports */
+import { OVM_ETH } from "../predeploys/OVM_ETH.sol";
+
+/**
+ * @title OVM_SequencerFeeWallet
+ * @dev Simple holding contract for fees paid to the Sequencer. Likely to be replaced in the future
+ * but "good enough for now".
+ *
+ * Compiler used: optimistic-solc
+ * Runtime target: OVM
+ */
+contract OVM_SequencerFeeWallet {
+
+    /*************
+     * Constants *
+     *************/
+
+    uint256 constant MIN_WITHDRAWAL_AMOUNT = 10 ether;
```

Comment:
This could be immuatable

Reply:
immutable is only used during constructors (immutable ~= constant but can be set during constructor)

Reply:
I see